File: src/java/xt/xt/audio/XtException.java
package xt.audio;

public final class XtException extends RuntimeException {
    private final long _error;
    public long getError() { return _error; }
    XtException(long error) { _error = error; }
    @Override public String getMessage() { return XtAudio.getErrorInfo(getError()).toString(); }
}

File: src/net/xt/XtException.cs
using System;

namespace Xt
{
	public sealed class XtException : Exception
	{
		readonly ulong _error;
		public ulong GetError() => _error;
		internal XtException(ulong error) => _error = error;
		public override string Message => XtAudio.GetErrorInfo(GetError()).ToString();
	}
}